modules/Bio/EnsEMBL/Compara/PipeConfig/EPO_conf.pm

```diff
@@ -67,7 +67,7 @@ sub default_options {
 
         # Executable parameters
         'mapping_params'    => { bestn=>11, gappedextension=>"no", softmasktarget=>"no", percent=>75, showalignment=>"no", model=>"affine:local", },
-        'enredo_params'     => ' --min-score 0 --max-gap-length 200000 --max-path-dissimilarity 4 --min-length 10000 --min-regions 2 --min-anchors 3 --max-ratio 3 --simplify-graph 7 --bridges -o ',
+        'enredo_params'     => ' --min-score 0 --max-gap-length 200000 --max-path-dissimilarity 4 --min-length 2000 --min-regions 2 --min-anchors 3 --max-ratio 3 --simplify-graph 7 --bridges -o ',
```

Comment:
I don't think this should be changed in the base class unless you've already discussed changing this param for _all_ EPOs? I would recommend creating a new plants-specific config (`modules/Bio/EnsEMBL/Compara/PipeConfig/Plants/EPO_conf`). If you do this, you will need to update the command you're setting in the JIRA ticket above.

Similarly, if you've changed it here, it should also be changed in `EPOwith2x_conf`.